packages/visx-xychart/src/components/series/BarSeries.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, useCallback, useMemo } from 'react';
+import { AxisScale } from '@visx/axis';
+import DataContext from '../../context/DataContext';
+import { SeriesProps } from '../../types';
+import withRegisteredData, { WithRegisteredDataProps } from '../../enhancers/withRegisteredData';
+import getScaledValueFactory from '../../utils/getScaledValueFactory';
+import isValidNumber from '../../typeguards/isValidNumber';
+import getScaleBandwidth from '../../utils/getScaleBandwidth';
+
+type BarSeriesProps<
+  XScale extends AxisScale,
+  YScale extends AxisScale,
+  Datum extends object
+> = SeriesProps<XScale, YScale, Datum> & {
+  /** Whether bars should be rendered horizontally instead of vertically. */
+  horizontal?: boolean;
+  /**
+   * Specify bar padding when bar thickness does not come from a `band` scale.
+   * Accepted values are [0, 1], 0 = no padding, 1 = no bar, defaults to 0.1.
```

Comment:
this is backwards and was updated to match d3's band scale logic: `1=no bar, 0=no padding`. will fix.